iphone_connector.py

```diff
@@ -132,15 +132,21 @@ def get_message_df():
     Returns:
         a pandas dataframe representing all text messages
     """
-    messages_df = pd.read_sql_query('''
-    SELECT
-      ROWID as message_id, text, handle_id, country, service, version,
-      DATETIME(date, 'unixepoch', '31 years') AS date,
-      DATETIME(date_read, 'unixepoch', '31 years') AS date_read,
-      DATETIME(date_delivered, 'unixepoch', '31 years') AS date_delivered,
-      is_emote, is_from_me, is_read, is_system_message, is_service_message, is_sent,
-      has_dd_results
-    FROM message''', _message_con)
+    try:
```

Comment:
Does this actually fix the issue?  I would've suspected an error to be thrown in the initialize method when we call sqlite3.connect()  If not, at the end of initialize() run a query such as `SELECT name FROM my_db.sqlite_master WHERE type='table';` on each database to confirm they're successfully connected.  It's much better to have initialize() fail instead of failed further downstream.

Also, although this may be an encryption error, all you're really catching is a database error.  As such, you should have a clearer error message, perhaps:

'A sqlite connection to the file at {0} failed, perhaps you've set iTunes to use encrypted backups?'.format(<file location>)